test/data/mock-send-state.json

```diff
@@ -1216,6 +1216,88 @@
       }
     ]
   },
+  "ramps": {
+    "buyableChains": [
+      {
+        "active": true,
+        "chainId": 1,
+        "chainName": "Ethereum Mainnet",
+        "nativeTokenSupported": true
```

Comment:
To keep the mock updated to the real values, please add the `shortName` property (here are the prod values https://on-ramp-content.api.cx.metamask.io/regions/networks)